projects/common/src/tokens/cookie.ts

```diff
@@ -0,0 +1,9 @@
+import {DOCUMENT} from '@angular/common';
+import {inject, InjectionToken} from '@angular/core';
+
+export const COOKIE = new InjectionToken<string>(
+    'An abstraction over document.cookie string',
+    {
+        factory: () => inject(DOCUMENT).cookie,
```

Comment:
My only concern is in Universal it might be `undefined`. Haven't checked that, but maybe we can just write `inject(DOCUMNT).cookie || ''`? And cover that by a test.